web/components/contract-search.tsx

```diff
@@ -227,115 +282,103 @@ export function ContractSearch(props: {
-                  {option.label}
-                </option>
-              ))}
-            </select>
-          )}
-        </Row>
-
-        {pillsEnabled && (
-          <Row className="scrollbar-hide items-start gap-2 overflow-x-auto">
-            <PillButton
-              key={'all'}
-              selected={pillFilter === undefined}
-              onSelect={selectPill(undefined)}
-            >
-              All
-            </PillButton>
+        {!hideOrderSelector && !query && (
+          <select
+            className="select select-bordered"
+            value={sort}
+            onChange={(e) => selectSort(e.target.value as Sort)}
+          >
+            {sortOptions.map((option) => (
+              <option key={option.value} value={option.value}>
+                {option.label}
+              </option>
+            ))}
+          </select>
+        )}
+      </Row>
+
+      {!additionalFilter && !query && (
+        <Row className="scrollbar-hide items-start gap-2 overflow-x-auto">
+          <PillButton
+            key={'all'}
+            selected={pillFilter === undefined}
+            onSelect={selectPill(undefined)}
+          >
+            All
+          </PillButton>
+          <PillButton
+            key={'personal'}
+            selected={pillFilter === 'personal'}
+            onSelect={selectPill('personal')}
+          >
+            {user ? 'For you' : 'Featured'}
+          </PillButton>
+
+          {user && (
             <PillButton
-              key={'personal'}
-              selected={pillFilter === 'personal'}
-              onSelect={selectPill('personal')}
+              key={'your-bets'}
+              selected={pillFilter === 'your-bets'}
+              onSelect={selectPill('your-bets')}
             >
-              {user ? 'For you' : 'Featured'}
+              Your bets
             </PillButton>
+          )}
 
-            {user && (
+          {pillGroups.map(({ name, slug }) => {
+            return (
               <PillButton
-                key={'your-bets'}
-                selected={pillFilter === 'your-bets'}
-                onSelect={selectPill('your-bets')}
+                key={slug}
+                selected={pillFilter === slug}
+                onSelect={selectPill(slug)}
               >
-                Your bets
+                {name}
               </PillButton>
-            )}
-
-            {pillGroups.map(({ name, slug }) => {
-              return (
-                <PillButton
-                  key={slug}
-                  selected={pillFilter === slug}
-                  onSelect={selectPill(slug)}
-                >
-                  {name}
-                </PillButton>
-              )
-            })}
-          </Row>
-        )}
-      </Col>
-
-      {filter === 'personal' &&
-      (follows ?? []).length === 0 &&
-      memberGroupSlugs.length === 0 ? (
-        <>You're not following anyone, nor in any of your own groups yet.</>
```

Comment:
I believe this was dead code, since if someone has no groups, we treat them as if they are subscribed to the default groups for the purposes of this querying code. So `memberGroupSlugs.length` would never be 0.

Reply:
Cool.